src/components/ThemeToggle/ThemeToggle.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { useTheme } from 'next-themes'
+import styles from './ThemeToggle.module.scss'
+
+const ThemeToggle = () => {
+  const { theme, setTheme } = useTheme()
+
+  return (
+    <button
+      type="button"
+      onClick={() => setTheme(theme === 'light' ? 'dark' : 'light')}
```

Comment:
issue: Looks like this check is reversed. If `theme` is set to `light` the theme should be set to `light mode`. This will set it to `dark` instead. Same issue on line 14. Since `light` should be the default might be better to write it:

```suggestion
      onClick={() => setTheme(theme === 'dark' ? 'dark' : 'light')}
```

Reply:
I disagree. Evaluating the current value of `theme`, if it is equal to `light` when the button is clicked, we want to set it to `dark`. If it the current value of `theme` does not equal `light` then it must equal `dark` so set `theme` to `light`

Reply:
it's confusing but it does work as expected. if the button says 'dark mode',the current value of `theme` is `light`. Click toggle -> `theme` is set to `dark`, and dark mode properly displays.

Reply:
That was my mistake you are totally right about it working correctly. I missed that it was in `onClick` method for some reason. My bad. Thank you for letting me know!